src/components/course/CourseSkills.jsx

```diff
@@ -1,30 +1,33 @@
 import React, { useContext, useState } from 'react';
 import { Badge, Button } from '@edx/paragon';
 
+import { AppContext } from '@edx/frontend-platform/react';
 import { CourseContext } from './CourseContextProvider';
 import { SKILLS_BUTTON_LABEL } from './data/constants';
 
 export const MAX_VISIBLE_SKILLS = 5;
 
 export default function CourseSkills() {
+  const { enterpriseConfig } = useContext(AppContext);
   const { state } = useContext(CourseContext);
   const { skillNames } = state.course;
   const [showMore, setShowMore] = useState(false);
   const skillsButtonLabel = showMore ? SKILLS_BUTTON_LABEL.SHOW_LESS : SKILLS_BUTTON_LABEL.SHOW_MORE;
-
   return (
     <div className="mb-5">
       <h5> Skills you&apos;ll gain</h5>
       <div>
         {skillNames.map((skill, index) => (
-          <Badge
-            key={skill.id}
-            className="course-skill"
-            variant="light"
-            style={{ display: ((index < MAX_VISIBLE_SKILLS) || showMore) ? 'inline-block' : 'none' }}
-          >
-            { skill }
-          </Badge>
+          <a href={`/${enterpriseConfig.slug}/search?skill_names=${skill}`}>
+            <Badge
```

Comment:
Would be nice to add the "pointer" cursor upon badge hover in the css here:

https://github.com/edx/frontend-app-learner-portal-enterprise/blob/master/src/components/course/styles/_CourseSkills.scss#L1

Reply:
@sameenfatima78 I have embedded Badge in the anchor tag so it will by default show the pointer cursor. We don't need to add anything here.